xml/System.Security.Claims/ClaimsPrincipal.xml

```diff
@@ -856,7 +856,13 @@ if (null != principal)
         <param name="match">The function that performs the matching logic.</param>
         <summary>Retrieves the first claim that is matched by the specified predicate.</summary>
         <returns>The first matching claim or <see langword="null" /> if no match is found.</returns>
-        <remarks></remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+Each <xref:T:System.Security.Claims.ClaimsIdentity> is called. <see also cref="M:System.Security.Claims.ClaimsIdentity.FindFirst(System.String)" />.
```

Comment:
Question for @rpetrusha: This is the first time I detect a "see also" tag in the remarks, but in this case, it seems it should be a simple xref link to the definition, preceded by the plain text "see also". Would you agree?:

```suggestion
Each <xref:T:System.Security.Claims.ClaimsIdentity> is called. See also: <xref:"M:System.Security.Claims.ClaimsIdentity.FindFirst(System.String)>.
```

Reply:
Same question applies for the "see also" cases below.

Reply:
Yes, I agree, @carlossanlop. Incidentally, when the crefs are transformed to xrefs, the type/member identifier is retained when it shouldn't be. So `<xref:T:System.Security.Claims.ClaimsIdentity>` is an invalid xref; it should be `<xref:System.Security.Claims.ClaimsIdentity>`.